mix.lock

```diff
@@ -1,3 +1,3 @@
 %{"earmark": {:hex, :earmark, "0.1.17"},
-  "hoedown": {:git, "git://github.com/hoedown/hoedown.git", "2a4cf17c70e6572ed42cdca3ea7ca3e768ed17be", []},
```

Comment:
You are using elixir master which now defaults to https. So you will need to be careful to not send the update mix.lock.